float32 voxelization
---
bool success